analysis_options.yaml

```diff
@@ -1,2 +1,215 @@
+# Updated according to
+# https://dart-lang.github.io/linter/lints/index.html
+# at 2020-08-10
+
 include: package:pedantic/analysis_options.yaml
 
+analyzer:
+  enable-experiment:
+    - non-nullable
```

Comment:
Can we disable this experiment?
It doesn't allow returning null values from methods.

Reply:
definitely, that's not available for Flutter yet anyway

Reply:
5fcf63eb4c20ecf7a2363430688dfa6ee977f093